service/src/main/kotlin/io/provenance/explorer/domain/models/explorer/ExplorerModels.kt

```diff
@@ -28,3 +40,37 @@ data class GasStatistics(
     val maxGasPrice: Int,
     val averageGasPrice: BigDecimal
 )
+
+data class Params(
+    val cosmos: CosmosParams,
+    val prov: ProvParams,
+)
+
+data class CosmosParams(
+    val authParams: AuthOuterClass.QueryParamsResponse,
```

Comment:
Youll need to create separate models for each of the different param types due to proto funkiness. I would also move them into a `ParamModels` file so they stay together and easily found.